Realm.podspec

```diff
@@ -30,7 +30,8 @@ Pod::Spec.new do |s|
                               'include/Realm/RLMRealm.h',
                               'include/Realm/RLMResults.h',
                               'include/Realm/RLMSchema.h',
-                              'include/Realm/Realm.h'
+                              'include/Realm/Realm.h',
+			      'include/Realm/RLMRealm_Dynamic.h'
```

Comment:
I think, we should be fine when our custom module map explicitly defines already a submodule for it. So no need for a subspec imo.